test/integration/sonobuoy_integration_test.go

```diff
@@ -125,6 +125,7 @@ func mustRunSonobuoyCommandWithContext(ctx context.Context, t *testing.T, ns, ar
 		} else {
 			t.Logf("Checked sonobuoy logs, got: %v", o.String())
 		}
+		t.FailNow()
```

Comment:
Noticed when testing locally this was missing. Will move it to its own PR though.